packages/frontend/package.json

```
@@ -13,11 +13,12 @@
     "@aws-sdk/s3-request-presigner": "3.627.0",
     "@aws-sdk/util-create-request": "3.622.0",
     "@aws-sdk/util-format-url": "3.609.0",
-    "@reach/router": "1.3.4",
+    "@reach/router": "^1.3.4",
     "p-event": "^6.0.1",
-    "react": "18.2.0",
+    "react": "^19.0.0",
```

Use fixed version of react dependencies.
```suggestion
    "react": "19.0.0",
```

When users try code from aws-samples, we want them to use a working version.